From 465abdedab4d10e598c8aa5f0ba85c1af343cfd1 Mon Sep 17 00:00:00 2001
From: Ryan Gonzalez <rymg19@gmail.com>
Date: Thu, 17 Dec 2020 16:54:05 -0600
Subject: [PATCH 2/3] Fix some format-security warnings

---
 tests/modules/config-policy.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/modules/config-policy.c b/tests/modules/config-policy.c
index 9e8dbb1..062c1fd 100644
--- a/tests/modules/config-policy.c
+++ b/tests/modules/config-policy.c
@@ -226,7 +226,7 @@ on_state_changed (WpEndpointLink *ep_link, WpEndpointLinkState old_state,
   case WP_ENDPOINT_LINK_STATE_ACTIVE:
     break;
   case WP_ENDPOINT_LINK_STATE_ERROR:
-    wp_message_object ("link failed: %s", error);
+    wp_message ("link failed: %s", error);
     g_test_fail ();
   case WP_ENDPOINT_LINK_STATE_INACTIVE:
   case WP_ENDPOINT_LINK_STATE_PREPARING:
-- 
2.29.2

